arch/arm64/arm64_litecov.cpp

```diff
@@ -61,6 +57,7 @@ void LiteCov::EmitCoverageInstrumentation(ModuleInfo *module,
 
   Arm64Assembler *arm64asm = static_cast<Arm64Assembler *>(assembler_);
 
+  skip_cov_instruction_br_off = module->instrumented_code_allocated;
```

Comment:
Nit: I would prefer to use a different (local) variable here, as skip_cov_instruction_br_off now sometimes holds the start offset and sometimes inserted code size. I don't think anything bad can happen now but might make it easier to introduce problems later.